src/Selenium2Library/keywords/_element.py

```diff
@@ -606,9 +606,11 @@ def page_should_not_contain_image(self, locator, message='', loglevel='INFO'):
 
     # Public, xpath
 
-    def get_matching_xpath_count(self, xpath):
+    def get_matching_xpath_count(self, xpath, return=str):
```

Comment:
Perhaps `return_str=True` would be better argument definition. 
1) It would not be confused with Python return statement. 
2) Then you could do something like this (in lines 625 - 628):
return str(count) if return_str else count